pkg/provider/azure/keyvault/keyvault_test.go

```diff
@@ -1447,7 +1447,7 @@ func TestAzureKeyVaultSecretManagerGetAllSecrets(t *testing.T) {
 			Attributes: &enabledAtt,
 		}
 
-		secretList := make([]keyvault.SecretItem, 1)
+		secretList := make([]keyvault.SecretItem, 0, 1)
```

Comment:
In fact it doesn't really matter since we are adding TWO items in the end down below. And then no other items are added to this list at all.

So you can change it to 

```go
		list := keyvault.SecretListResult{
			Value: &[]keyvault.SecretItem{secretItemOne, secretItemTwo},
		}
```

And while you're at it, you can change all the rest of the list value assignments as well. :)